assignments/objects.js

```diff
@@ -19,46 +19,161 @@ const example = {
 
 // Write your intern objects here:
 
+const mitzi = {
 
-// ==== Challenge 2: Reading Object Data ==== 
+  id: 1,
+  name: "Mitzi",
+  email: "mmelloy0@psu.edu",
+  gender: "F"
+
+}
+
+const kennan = {
+
+  id: 2,
+  name: "Kennan",
+  email: "kdiben1@tinypic.com",
+  gender: "M",
+  speak: function() {
+
+    return "Hello, my name is " + this.name + "!";
```

Comment:
You could use [template literal syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here:
```javascript
return `Hello, my name is ${this.name}!`;
```